sceptre/plan/actions.py

```diff
@@ -784,9 +793,11 @@ def timed_out(elapsed):
 
         status = StackStatus.IN_PROGRESS
 
-        self.most_recent_event_datetime = (
-            datetime.now(tzutc()) - timedelta(seconds=3)
-        )
```

Comment:
I'm confused here. It looks like you added this property to this class... but then this diff seems to indicate it was already here... were we adding it where it wasn't declared in `__init__` before?

Reply:
That's correct. This value wasn't being initialized previously in __init__.